code/script.js

```diff
@@ -1,51 +1,117 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
+let botQuestion = 1
 
```

Comment:
It's good to have the variables declared at the top of the page, I've heard, so this is good 👍